custom/icds_reports/scripts/migrate_tables.py

```diff
@@ -262,7 +262,7 @@ def get_table_sizes(source_db, source_host, source_user):
     try:
         from sqlalchemy import create_engine
     except ImportError:
-        print('sqlalchemy not installed. Progress not supported.')
+        print('\nsqlalchemy not installed. Progress not supported.\n')
```

Comment:
You can also put in escape codes to make it red if you like, similar to https://github.com/dimagi/commcare-hq/blob/acd5f7c925e6508dab376c3a248a89118dd2e027/scripts/rebuildstaging.py#L349-L360